jsk_unitree_robot/cross/docker/Dockerfile_ros1

```diff
@@ -29,7 +29,9 @@ RUN apt install -y --no-install-recommends $(cat ros-packages.txt | grep -v ^#)
 RUN apt install -y python-pip
 RUN git clone https://github.com/k-okada/rosinstall_generator /tmp/rosinstall_generator -b add_depend_type
 RUN pip install /tmp/rosinstall_generator
-RUN apt install -y python3-vcstool
+RUN apt install -y python3-vcstool python-typing
+# python-typing is necessary to avoid import error in build_ros1_dependencies.sh
```

Comment:
The basic idea of this Docker files is to build same environment as a real Unitree robot, except for development tools that are only needed to build  theenvronment, such as rosintall or vcstool. Therefore, installing python-typing here will not work on a real environment unless they alredy have one.